script/build.mjs

```diff
@@ -0,0 +1,52 @@
+import assert from 'node:assert/strict'
+import fs from 'node:fs/promises'
+import {fetch} from 'undici'
```

Comment:
could native/expiremental fetch be used directly?

Reply:
per above: doesn’t work in TS as far as I know

Reply:
per above, this file isn't type checked

Reply:
check now!